modules/standard/Random.chpl

```diff
@@ -624,7 +760,7 @@ module Random {
 
     :return: A zero-based array of ``n`` random values sampled from the range
 
-    :throws IllegalArgumentError: If ``n < 1`` or if ``n > arr.size`` and
+    :throws IllegalArgumentError: If ``n < 1`` or if ``n > r.size`` and
```

Comment:
I would mention in your PR message that you fixed the documentation for these functions